Enterprise-CMCS/macpro-mdct-mfp
# MDCT-2759 + 2760: Transition Benchmarks w/ Drawer

### Description

Creating the `Transition Benchmarks` page with its corresponding drawer

### Related ticket(s)


MDCT-2759
MDCT-2760

---
### How to test

- Log in as a state user
- Navigate to `/wp/transition-benchmarks`

You should see the corresponding data and styling!

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
